experiments/utils/download.py

```diff
@@ -31,14 +31,14 @@ def create_download_response_zip(file_format: str, experiment: Experiment) -> \
     if file_format == 'raw':
         zip_file = _create_zip(
             experiment,
-            lambda dp: str(dp.number) + '.txt',
+            lambda dp: str(dp.session.subject_id) + '.txt',
             # Raw should just return the data of the DataPoint
```

Comment:
Could you look into adding a DP specific identifier to the filename as well?
Otherwise we'd get name collisions if multiple datapoints are added to a session

Reply:
Yes, sure i'll add it.

Reply:
Should be fixed in: [4f985a7](https://github.com/UiL-OTS-labs/web-experiment-datastore/pull/39/commits/4f985a78986dcbb4bed1ab5d9fc402d626dc8f73)